FinnUI/Sources/MyVehicles/MyVehiclesListView/MyVehiclesListView.swift

```diff
@@ -0,0 +1,118 @@
+import SwiftUI
+
+@available(iOS 13.0.0, *)
+struct MyVehiclesListView: View {
+
+    // MARK: - Private
+
+    @State var navigateToPreviouslyOwnedVehicles = false
+    private let imageUrl = "https://images.finncdn.no/dynamic/default/"
+
+    // MARK: - Public
+
+    var viewModel: MyVehiclesListViewModel
+    var onTapAddNewVehicle: (() -> Void) = {}
+
+    var body: some View {
+        NavigationView {
+            List {
+                addNewVehicle
+                ForEach(viewModel.vehicles, id: \.self) { viewModel in
+                    NavigationLink(destination: EmptyView()) {
+                        MyVehicleCell(
+                            viewModel: viewModel,
+                            imageProvider: SampleSingleImageProvider(
+                                url: self.constructURL(imagePath: viewModel.imagePath))
+                        )
+                    }
+                }
+                Group {
+                    seePreviouslyOwnedVehicles
+                    NavigationLink(destination: EmptyView(), isActive: $navigateToPreviouslyOwnedVehicles) {
+                        EmptyView()
+                    }
+                }
+            }
+            .appearance { (view: UITableView) in
+                view.separatorStyle = .none
+                view.backgroundColor = .bgPrimary
+            }
+            .appearance { (view: UITableViewCell) in
+                view.selectionStyle = .none
+            }
+            .navigationBarTitle("\(viewModel.title)", displayMode: .inline)
```

Comment:
huh? viewModel.title is a string already

Reply:
This one is a big "huh?" because the compiler is complaining :P
![Screenshot 2020-09-01 at 15 18 22](https://user-images.githubusercontent.com/8507719/91856144-714e5600-ec66-11ea-8365-c17437971a19.png)